.buildkite/heartbeat/heartbeat-pipeline.yml

```diff
@@ -36,7 +39,9 @@ steps:
           - "heartbeat/build/*.json"
```

Comment:
can we add per-step notifications as done in other PRs (also applies to several steps below)? see https://github.com/elastic/beats/blob/main/.buildkite/metricbeat/pipeline.yml#L42-L44